src/clients/mocks/MockSpokePoolClient.ts

```diff
@@ -92,17 +92,20 @@ export class MockSpokePoolClient extends SpokePoolClient {
           events[idx].push(event);
         }
       });
+    const bnMax = (val: BigNumber, cmp: BigNumber) => {
+      return val.gt(cmp) ? val : cmp;
+    };
```

Comment:
I thought we had this somewhere in `BNUtils` or something - but I can't find it anywhere. It'd be good to put it somewhere common because I'm sure it's not the last time we'll use it.

Reply:
Moved: https://github.com/across-protocol/sdk/pull/830/commits/20d698c8a610ed4b37e6976a1dd971816338ff7e